guides/source/active_record_migrations.md

```diff
@@ -346,20 +373,23 @@ class AddDetailsToProducts < ActiveRecord::Migration[8.0]
 end
 ```
 
-TIP: Have a look at the generators help output (`bin/rails generate --help`)
-for further details.
+TIP: For further help with generators, run `bin/rails generate --help`.
+Alternatively, you can also run `bin/rails generate model --help` or `bin/rails
+generate migration --help` for help with specific generators.
 
-Writing Migrations
+Updating Migrations
 ------------------
 
-Once you have created your migration using one of the generators it's time to
-get to work!
+Once you have created your migration file using one of the generators from the
+above [section](#generating-migration-files), you can update the generated
+migration file in the `db/migrate` folder to define further changes you want to
+make to your database schema.
 
 ### Creating a Table
 
-The [`create_table`][] method is one of the most fundamental, but most of the time,
-will be generated for you from using a model, resource, or scaffold generator. A typical
-use would be
+The [`create_table`][] method is one of the most fundamental, but most of the
```

Comment:
The most fundamental ... seems like we're missing a word. And by fundamental would be it more clear to say most "common additions" (adding a table).